src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawPanelControllerAdapter.java

```diff
@@ -4,14 +4,15 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
- * driver adapter to drawer with several bugs.
+ * driver adapter to drawer with several bugs fixed.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawPanelControllerAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it lacks a driver aspect.

Inheritance is not used (no super drawLine calls). Should be removed!